common-rendering/package.json

```diff
@@ -7,7 +7,11 @@
     },
     "dependencies": {
         "@emotion/react": "^11.4.1",
+        "@guardian/src-button": "^3.8.0",
         "@guardian/src-foundations": "^3.9.0",
+        "@guardian/src-icons": "^3.8.0",
+        "@guardian/src-link": "^3.8.0",
+        "@guardian/types": "^8.0.0",
```

Comment:
it's worth noting that the [types library](https://github.com/guardian/types/blob/main/README.md) has been deprecated. It got replaced by `@guardian/libs`

Reply:
Ah OK, I'll take a note to change it and use guardian/libs in the coming coming PRs. Thanks for letting us know